crates/solver/src/settlement_ranker.rs

```diff
@@ -208,6 +210,32 @@ impl SettlementRanker {
             });
         }
 
+        // Filter out settlements with too high score.
+        if Utc::now() > self.auction_rewards_activation_timestamp {
```

Comment:
I think at this point we no longer have to feature gate CIP-20 stuff, right?

Reply:
Yes, but wanted to keep it for consistency since the option of reverting the CIP20 changes is still there (at least theoretically :) )